Portal/View/Components/TextField.swift

```diff
@@ -119,12 +119,12 @@ internal struct TextFieldChangeSet<MessageType> {
     
     static func fullChangeSet(
         properties: TextFieldProperties<MessageType>,
-        styleSheet: StyleSheet<TextFieldStyleSheet>,
+        style: StyleSheet<TextFieldStyleSheet>,
```

Comment:
Why did you rename this?

Reply:
To be consistent with others components like Button